docs/scheduler-perf/scheduler-perf.md

```diff
@@ -0,0 +1,856 @@
+cd ..
+make crc_storage
+make input
+make openstack
+make openstack_deploy
+```
+
+Then cinder was configured to use LVM with iSCSI and 10 cinder API services (to
+try to remove possible API performance bottlenecks) by editing the existing
+`openstackcontrolplane`:
+
+```bash
+$ oc edit openstackcontrolplane
+```
+
+And replacing the `cinder` section with the following (the `cinderBackup`
+section is irrelevant for these tests, but that's what was used):
+```yaml
+  cinder:
+    enabled: true
+    template:
+      cinderAPI:
+        containerImage: quay.io/podified-antelope-centos9/openstack-cinder-api:current-podified
+        customServiceConfig: |
+          [DEFAULT]
+          log_file=
+          service_down_time=180
+          debug=False
+        debug:
+          service: false
+        externalEndpoints:
+        - endpoint: internal
+          ipAddressPool: internalapi
+          loadBalancerIPs:
+          - 172.17.0.80
+          sharedIP: true
+          sharedIPKey: ""
+        replicas: 10
+        resources: {}
+      cinderBackup:
+        containerImage: quay.io/podified-antelope-centos9/openstack-cinder-backup:current-podified
+        customServiceConfig: |
+          [DEFAULT]
+          backup_driver=cinder.backup.drivers.nfs.NFSBackupDriver
+          backup_share=192.168.1.2:/Public
+        debug:
+          service: false
+        networkAttachments:
+        - storage
+        replicas: 1
+        resources: {}
+      cinderScheduler:
+        containerImage: quay.io/podified-antelope-centos9/openstack-cinder-scheduler:current-podified
+        customServiceConfig: |
+          [DEFAULT]
+          debug=False
+        debug:
+          service: false
+        replicas: 1
+        resources: {}
+      cinderVolumes:
+        lvm-iscsi:
+          containerImage: quay.io/podified-antelope-centos9/openstack-cinder-volume:current-podified
+          customServiceConfig: |
+            [DEFAULT]
+            debug = False
+            [lvm]
+            max_over_subscription_ratio=10000000
+            image_volume_cache_enabled=false
+            volume_driver=cinder.volume.drivers.lvm.LVMVolumeDriver
+            volume_group=cinder-volumes
+            target_protocol=iscsi
+            target_helper=lioadm
+            volume_backend_name=lvm_iscsi
+          debug:
+            service: true
+          networkAttachments:
+          - storage
+          replicas: 1
+          resources: {}
+      customServiceConfig: |
+        [DEFAULT]
+        #service_down_time=180
+        debug=True
+      databaseInstance: openstack
+      databaseUser: cinder
+      debug:
+        dbSync: false
+      passwordSelectors:
+        database: CinderDatabasePassword
+        service: CinderPassword
+      preserveJobs: false
+      rabbitMqClusterName: rabbitmq
+      secret: osp-secret
+      serviceUser: cinder
+```
+
+The cinder volume service was setup to not start automatically to allow us to
```

Comment:
It took me a moment to figure out what you meant, and I now see that the statement is true because of what's on L349. I think this could be made more explicit (but I don't have a specific recommendation on how to do that).